examples/diffusers/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers
+diffusers
```

Comment:
@jagadeeshi2i the diffusers package is evolving pretty rapidly and the api is still very fluent. Could we fix the revision to make sure the example still works after some time?